### PR TITLE
chore(Autolayout): doc update

### PR DESCRIPTION
**Pull request by Robert-Louis**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

- Did not reflet the change from control to panel 

## What is the new behavior?

- counter and primary lenght are no longer deprecated

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/controls/AutoLayoutControl.md [hunk 1]
```diff
@@ -46,6 +46,6 @@ Property|Type|Description
 -|-|-
 PrimaryAlignment|AutoLayoutPrimaryAlignment|Either Auto, the child grid take only the place it need in the autolayout direction, or Stretch, the child grid take all the available place in the autolayout direction. Note if set to Strech AutoLayoutJustify will behave as if in Stack mode.
 CounterAlignment|AutoLayoutAlignment|Indicates where an element should be displayed on the counter axis of the parent orientation relative to the allocated layout slot of the parent element.
-PrimaryLength|double| Set the grid size in parent orientation [Deprecated should Height and Width instead].
-CounterLength|double|Sets the child size in parent counter orientation [Deprecated should Height and Width instead]. 
+PrimaryLength|double| Set the grid size in parent orientation. Height or Width (depending of the Orientation) will have priority.
```
Comment (kazo0) [inline doc/controls/AutoLayoutControl.md:49]:
```suggestion
PrimaryLength|double| Sets the height or width of the child depending on the `Orientation`.
```

File: doc/controls/AutoLayoutControl.md [hunk 2]
```diff
@@ -46,6 +46,6 @@ Property|Type|Description
...
+CounterLength|double|Sets the child size in parent counter orientation. Height or Width (depending of the Orientation) will have priority.
```
Comment (kazo0) [inline doc/controls/AutoLayoutControl.md:50]:
```suggestion
CounterLength|double|Sets the height or width of the child depending on the inverse of `Orientation`.
```

Comment (kazo0) [reply doc/controls/AutoLayoutControl.md:50]: I _think_ this is how it should be written, unless I'm misunderstanding how CounterLength works

File: doc/controls/AutoLayoutControl.md (hunk 1 above)
Comment (kazo0) [reply doc/controls/AutoLayoutControl.md:49]: I _think_ this is how it should be written, unless I'm misunderstanding how PrimaryLength works

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**